sympy/combinatorics/perm_groups.py

```diff
@@ -678,6 +680,132 @@ def basic_transversals(self):
+            transversals[l] = sorted(t.values(), key = lambda x: base_ordering[base[l]^x])
+
+        orbits = H.basic_orbits
+        h_stabs = H.basic_stabilizers
+        g_stabs = self.basic_stabilizers
+
+        indices = [x.order()//y.order() for x, y in zip(g_stabs, h_stabs)]
+
+        # T^(l) should be a right transversal of H^(l) in G^(l) for 1<=l<=len(base)
+        # While H^(l) is the trivial group, T^(l) contains all the elements of G^(l)
+        # so we might just as well start with l = len(h_stabs)-1
+        T = g_stabs[len(h_stabs)]._elements
+        t_len = len(T)
+        l = len(h_stabs)-1
+        while l > -1:
+            T_next = []
+            for u in transversals[l]:
+                if u == identity:
+                    continue
+                b = base_ordering[base[l]^u]
+                for t in T:
+                    p = t*u
+                    if all([base_ordering[h^p] >= b for h in orbits[l]]):
+                        T_next.append(p)
+                    if t_len + len(T_next) == indices[l]:
+                        break
+                if t_len + len(T_next) == indices[l]:
+                    break
+            T += T_next
+            t_len += len(T_next)
+            l -= 1
+        return T
+
+    def _coset_representative(self, g, H):
+        """Return the representative of gH from the transversal that
+        would be computed by `self.coset_transversal(H)`.
+
+        """
+        if H.order() == 1:
+            return g
+        #The base of self must be an extension of H.base.
+        if not(self.base[:len(H.base)] == H.base):
+            self._schreier_sims(base=H.base)
+        orbits = H.basic_orbits[:]
+        h_transversals = [list(_.values()) for _ in H.basic_transversals]
+        transversals = [list(_.values()) for _ in self.basic_transversals]
+        base = self.base
+        base_ordering = _base_ordering(base, self.degree)
+        def step(l, x):
+            gamma = sorted(orbits[l], key = lambda y: base_ordering[y^x])[0]
+            i = [base[l]^h for h in h_transversals[l]].index(gamma)
+            x = h_transversals[l][i]*x
+            if l < len(orbits)-1:
+                for u in transversals[l]:
+                    if base[l]^u == base[l]^x:
+                        break
+                x = step(l+1, x*u**-1)*u
+            return x
+        return step(0, g)
+
+    def coset_table(self, H):
+        """Return the standardised (right) coset table of self in H as a list of lists.
+        """
+        #Maybe this should be made to return an instance of CosetTable from fp_groups.py
+        #but the class would need to be changed first to be compatible with PermutationGroups
+
+        from itertools import chain, product
+        if not H.is_subgroup(self):
+            raise ValueError("The argument must be a subgroup")
+        T = self.coset_transversal(H)
+        n = len(T)
+
+        A = list(chain.from_iterable((gen, gen**-1) \
```

Comment:
Backslash continuations should preferably be avoided whenever possible. They are typically necessary at breaks after == but not if there are open parentheses or brackets to be closed later. (This also applies to the unit tests.)